File: single-language-analysis/dataset-files/java/TestSmell084.java
// Original URL: https://github.com/lijtforgithub/study-spring/blob/a9016a4b8be9202a7b3071879fea719c2531136e/src/main/resources/old/activemq/JMSTest.java#L43-L50

public class TestSmell084 {
    @Test
    public void testTopic() {  
    	this.applicationContext = TestUtils.instanceContext(MODULE, CONFIG_LISTENER);
    	ProducerService producerService = this.applicationContext.getBean(ProducerService.class, "producerService");
    	Destination queue = this.applicationContext.getBean("topic", Destination.class);
    	
    	producerService.sendMessage(queue, "LiJingTang-JMS-Message-Topic");  
    }
}

File: single-language-analysis/dataset-files/python/test_smell_084.py
import pytest

class TestJmstest:
    def test_smell_084(self):
        self.application_context = TestUtils.instance_context(MODULE, CONFIG_LISTENER)
        producer_service = self.application_context.get_bean(ProducerService, "producerService")
        queue = self.application_context.get_bean("topic", Destination)
        
        producer_service.send_message(queue, "LiJingTang-JMS-Message-Topic")